frontend/src/assets/images/draft-parcel-icon.svg

```diff
@@ -1 +1 @@
-<svg version="1.2" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" width="40" height="40"><g><path xmlnsDefault="http://www.w3.org/2000/svg" d="M18 8c0-3.31-2.69-6-6-6S6 4.69 6 8c0 4.5 6 11 6 11s6-6.5 6-11zm-8 0c0-1.1.9-2 2-2s2 .9 2 2-.89 2-2 2c-1.1 0-2-.9-2-2zM5 20v2h14v-2H5z" style="fill: rgb(46, 133, 64);" vector-effect="non-scaling-stroke"/></g></svg>
```

Comment:
Curious what the reason for removing?

Reply:
Umm... I'll look into this, I corrected an svg but don't remember removing one.